src/main/php/xp/web/srv/Develop.class.php

```diff
@@ -21,6 +21,9 @@ class Develop extends Server {
    * @param  string[] $logging
    */
   public function serve($source, $profile, $webroot, $docroot, $config, $args, $logging) {
+    $environment= new Environment($profile, $webroot, $docroot, $config, $args, $logging);
+    $application= (new Source($source, $environment))->application($args);
+    $application->initialize();
```

Comment:
We should only run this once per server run, e.g. by using a tempfile with the PID.